sql/migrations/20210303085717_world.sql

```diff
@@ -44,6 +44,9 @@ UPDATE `gameobject_loot_template` SET `maxcount` = 3 WHERE `entry` = 1415 AND `i
 UPDATE `spell_affect` SET `SpellFamilyMask` = 6447547418 WHERE `entry` = 18218; 
 UPDATE `spell_affect` SET `SpellFamilyMask` = 6447547418 WHERE `entry` = 18219;
 
+-- Marsh Flesheater and Bloodscalp Berserker should not flee at low hp
+DELETE FROM `creature_ai_scripts` WHERE `id` IN (59702, 75102);

```

Comment:
You must delete the event too, not just the script.

Reply:
There are other scripts assigned to that event.